chemprop/cli/train.py

```diff
@@ -1028,3 +1058,61 @@ def main(args):
     logging.basicConfig(stream=sys.stdout, level=logging.DEBUG, force=True)
     args = parser.parse_args()
     TrainSubcommand.func(args)
+
+
+def summarize(task_type: str, dataset: MoleculeDataset) -> tuple[list, list]:
```

Comment:
```suggestion
def summarize(task_type: str, dataset: _MolGraphDatasetMixin) -> tuple[list, list]:
```
It can also be ReactionDatasets. It just needs to have y values.